cypress/integration/plugins/observability-dashboards/trace_analytics_traces.spec.js

```diff
@@ -6,6 +6,10 @@
 /// <reference types="cypress" />
 
 import { delayTime, setTimeFilter, SPAN_ID, TRACE_ID } from '../../../utils/constants';
+import customParseFormat from 'dayjs/plugin/customParseFormat';
+import dayjs from 'dayjs';
```

Comment:
hi @ps48 , I remember dayjs is not supported in Cypress 5.6.0, right?

Reply:
yes, @derek-ho For branches 1.3 and prior, FTR uses cypress 5 which doesn't have default shipped moment.js rather it has moment.js. Can you edit these changes to use moment.js? Here's the related PR: https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/515